js/artFeature.js

```diff
@@ -17,3 +23,23 @@ const creatingArtFromHtml = () => {
 document
   .querySelector('#html-code')
   .addEventListener('keyup', creatingArtFromHtml);
+
+instructionButton.onclick = () => {
+  instructionModal.style.display = overlay.style.display = 'block';
+  document.querySelector('body').style.overflow = 'hidden';
+  if (mobileScreen.matches) {
+    instructionModal.style.height = '350px';
+  }
+};
+
+closeButton.onclick = () => {
+  instructionModal.style.display = overlay.style.display = 'none';
```

Comment:
If you are handling modal with display: none you cannot do transitions. How are you handling the transition of open and close of modal?

Reply:
I haven't used any transition for open/close of modal. Should I add it?